data-integration/main.bal

```diff
@@ -0,0 +1,102 @@
+import data_integration.store;
+import ballerina/http;
+import ballerina/persist;
+
+configurable int port = 9090;
+
+service /taskmanager on new http:Listener(port) {
+    private final store:Client dbClient;
+
+    function init() returns error? {
+        self.dbClient = check new();
+    }
+
+    isolated resource function post employee(store:EmployeeInsert employee)
```

Comment:
Don't have to explicitly specify `isolated`. Just need to make sure the code meets the requirements for inferring `isolated`.